tests/base/test_tensor_innerproduct.py

```diff
@@ -162,6 +162,166 @@ def test_order6_faces_invert_model(self):
         self.orderTest()
 
 
+class TestInnerProductsFaceProperties3D(discretize.tests.OrderTest):
+    """Integrate a function over a surface within a unit cube domain
+    using edgeInnerProducts and faceInnerProducts."""
+
+    meshTypes = ["uniformTensorMesh"]
+    meshDimension = 3
+    meshSizes = [16, 32]
+
+    def getError(self):
+        call = lambda fun, xyz: fun(xyz[:, 0], xyz[:, 1], xyz[:, 2])  # NOQA E731
+
+        ex = lambda x, y, z: x**2 + y * z  # NOQA E731
```

Comment:
Error E731 is already ignored in all of discretize's tests. All of the ` # NOQA E731` in here are unnecessary.